src/components/Sidebar.js

```diff
@@ -170,14 +172,15 @@ function filterBuckets(buckets, filters): BucketEntry[] {
 }
 
 type BucketsMenuState = {
+  canCreateBucket: boolean,
   hideReadOnly: boolean,
   search: ?string,
 };
 
 class BucketsMenu extends PureComponent<BucketsMenuProps, BucketsMenuState> {
   constructor(props: BucketsMenuProps) {
     super(props);
-    this.state = { hideReadOnly: false, search: null };
+    this.state = { canCreateBucket: true, hideReadOnly: false, search: null };
```

Comment:
This might be a dumb question but why do we need to put it in the state?

Reply:
Good catch! I may have done too much while making Flow happy :grin: